cli/valet.php

```diff
@@ -248,6 +248,17 @@
 })->descriptions('Determine if this is the latest version of Valet');
 
 /**
+ * Switch between versions of PHP
+ */
+$app->command('switch-to [php_version]', function ($php_version) {
```

Comment:
Let's change this to `valet use php71` or similar, using the actual homebrew package name so we don't have to do anything fancy.